core/modules/catalog/helpers/getProductGallery.ts

```diff
@@ -15,7 +15,7 @@ export default function getProductGallery (product: Product) {
   }
 
   const productGallery = uniqBy(configurableChildrenImages(product).concat(getMediaGallery(product)), 'src')
-    .filter(f => f.src && f.src !== config.images.productPlaceholder)
+    .filter(f => f.src)
```

Comment:
why you removed that?

Reply:
Actually, this condition does not allow a placeholder to be in the productGallery, and hence to add a placeholder in the gallery removed this condition.